ansible/roles/bootstrap/tasks/main.yml

```diff
@@ -0,0 +1,44 @@
+- name: Ensures bootkube dir exists
+  file: path=/tmp/bootkube state=directory
+
+- name: Extract bootkube binaries
+  unarchive:
+    src: https://github.com/kubernetes-incubator/bootkube/releases/download/v0.3.9/bootkube.tar.gz
+    dest: /tmp/bootkube
+    remote_src: True
+
+- name: Render bootkube manifests
+  command: /tmp/bootkube/bin/linux/bootkube render --asset-dir=/tmp/bootkube/assets --experimental-self-hosted-etcd --etcd-servers=http://10.3.0.15:2379 --api-servers=https://{{ inventory_hostname }}:443
```

Comment:
@alanmeadows I was able to validate the approach with Jan of using a DNS FQDN providing multiple IPs for a single domain to provide load balancing.  In my approach, I modified the `inventroy_hostname` to be another variable, `kube_master_fqdn` that is provided instead.   Once this PR gets merged, I'll submit a PR with my modification. Currently, @janwillies and I are looking into ways to self-host the DNSMasq server we tested with.